Failure in TestSourcesTest.test_longStream

http://jenkins.hazelcast.com/job/jet-oss-pr-builder/3163/testReport/junit/com.hazelcast.jet.pipeline.test/TestSourcesTest/test_longStream_1__mode_client_/

Stacktrace:
```
java.lang.AssertionError: 

Expected: exception with message a string containing "com.hazelcast.jet.pipeline.test.AssertionCompletedException"
     but: message was "Exception in ProcessorTasklet{assertCollectedEventually#0}: java.lang.AssertionError: java.lang.AssertionError: Invalid items per second"
Stacktrace was: com.hazelcast.jet.JetException: Exception in ProcessorTasklet{assertCollectedEventually#0}: java.lang.AssertionError: java.lang.AssertionError: Invalid items per second
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:373)
	at java.util.concurrent.CopyOnWriteArrayList.forEach(CopyOnWriteArrayList.java:891)
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.run(TaskletExecutionService.java:346)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.AssertionError: java.lang.AssertionError: Invalid items per second
	at com.hazelcast.jet.pipeline.test.AssertionSinks$CollectingSinkWithTimer.timer(AssertionSinks.java:214)
	at com.hazelcast.function.ConsumerEx.accept(ConsumerEx.java:46)
	at com.hazelcast.jet.impl.pipeline.test.AssertionP.maybeFireTimer(AssertionP.java:67)
	at com.hazelcast.jet.impl.pipeline.test.AssertionP.tryProcess(AssertionP.java:60)
	at com.hazelcast.jet.impl.processor.ProcessorWrapper.tryProcess(ProcessorWrapper.java:103)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:281)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.stateMachineStep(ProcessorTasklet.java:266)
	at com.hazelcast.jet.impl.execution.ProcessorTasklet.call(ProcessorTasklet.java:240)
	at com.hazelcast.jet.impl.execution.TaskletExecutionService$CooperativeWorker.runTasklet(TaskletExecutionService.java:366)
	... 3 more

```

--- a/hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/test/TestSourcesTest.java
+++ b/hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/test/TestSourcesTest.java
@@ -51,7 +51,7 @@ public class TestSourcesTest extends PipelineTestSupport {
 
     @Test
     public void test_longStream() throws Throwable {
-        int itemsPerSecond = 1_000;
+        int itemsPerSecond = 250;
         int timeoutSeconds = 10;
         int numWindowResultsToWaitFor = 3;
 

